k8s/aat/common/reform-scan/blob-router.yaml

```diff
@@ -25,6 +34,43 @@ spec:
         STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES: 0
         CRIME_ENABLED: "true"
         INCLUDE_WRITE_PERMISSION_IN_STORAGE_SAS_TOKEN: false
+      testsConfig:
+        memoryLimits: "3072Mi"
+        serviceAccountName: tests-service-account
+        keyVaults:
+          reform-scan:
+            resourceGroup: reform-scan
+            secrets:
+              # Base64-encoded PKCS 12 key store, containing client's private key and certificate known to the API
+              test-valid-key-store: TEST_VALID_KEY_STORE
+              test-valid-key-store-password: TEST_VALID_KEY_STORE_PASSWORD
+              # Base64-encoded PKCS 12 key store containing private key and certificate known to the API, but expired
+              test-expired-key-store: TEST_EXPIRED_KEY_STORE
+              test-expired-key-store-password: TEST_EXPIRED_KEY_STORE_PASSWORD
+              # Base64-encoded PKCS12 key store containing private key and certificate known to the API, but not yet valid
+              test-not-yet-valid-key-store: TEST_NOT_YET_VALID_KEY_STORE
+              test-not-yet-valid-key-store-password: TEST_NOT_YET_VALID_KEY_STORE_PASSWORD
+              test-subscription-key: TEST_SUBSCRIPTION_KEY
+              storage-account-secondary-key: TEST_STORAGE_ACCOUNT_KEY
+              storage-account-name: TEST_STORAGE_ACCOUNT_NAME
+        environment:
+          TEST_URL: "http://reform-scan-blob-router-aat.service.core-compute-aat.internal"
+          SLACK_CHANNEL: "bsp-test-notices"
+          SLACK_NOTIFY_SUCCESS: true
+          TEST_STORAGE_CONTAINER_NAME: "bulkscan"
+          TEST_STORAGE_ACCOUNT_URL: "https://reformscanaat.blob.core.windows.net"
+          API_GATEWAY_URL: "https://core-api-mgmt-aat.azure-api.net/reform-scan"
```

Comment:
should it not be platform url instead?

Reply:
it is for cert test. current url is like this in cnp jenkins